### DISCUSSION TITLE
Error saloon post request

### DISCUSSION THREAD

**Discussion by AbrorGOF** [saloonphp/saloon#125] [Q&A]
Hi everyone when I send post request Saloon returned this error

```
strtolower(): Argument #1 ($string) must be of type string, int given
vendor\guzzlehttp\psr7\src\Utils.php: 24
```

Help needed if anyone has fixed this

**Comment by Sammyjo20** [comment]
Hey @AbrorGOF 

Would you mind sharing your request and connector please and how you are making the request.

Thanks!

**Reply by AbrorGOF** [reply]
Thanks a lot for your response!
I have found my error. I have remove header in connector it was the reason of error.
![image](https://user-images.githubusercontent.com/55149378/207118172-f84e56fa-3c84-4a93-97d9-0dd507ba4829.png)

**Reply by Sammyjo20** [reply]
Ah, you need to change the header to:

‘Content-Type’ => ‘value’ instead :)
